scripts/golangci-lint.yml

```diff
@@ -27,6 +27,6 @@ jobs:
         run: sudo apt-get update && sudo apt-get -y install libsnmp-dev
         if: github.repository == 'prometheus/snmp_exporter'
       - name: Lint
-        uses: golangci/golangci-lint-action@v3.4.0
```

Comment:
Can you remove this change ? It's done via dependabot and we just upgrade it to the v3.7.0

Reply:
Done - removed.